R/bernoulli.R

```diff
@@ -157,3 +157,17 @@ cdf.bernoulli <- function(d, x, ...) {
 quantile.bernoulli <- function(d, p, ...) {
   qbinom(p = p, size = 1, prob = d$p)
 }
+
+#' Fit a bernoulli distribution to data
+#'
+#' @param d a `bernoulli` object
+#' @param x a vector of zeroes and ones to fit the bernoulli distribution to
+#'
+#' @return a `bernoulli` object
+#' @export
+fit.bernoulli <- function(d, x) {
+  x <- as.integer(x)
+  valid_x <- all(x %in% c(0L, 1L))
```

Comment:
Checking for integer class is probably too strict.

Reply:
I've removed the integer conversion from `fit.bernoulli` and `fit.binomial`. Do you want any input checking? There's none for checking probabilities passed into `quantile` for example so I'm going by that.

Reply:
Maybe down the line? Let's not worry about it just yet.